pkg/ibmvpc/parser_test.go

```diff
@@ -0,0 +1,68 @@
+package ibmvpc
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/np-guard/models/pkg/ipblocks"
+	"github.com/np-guard/vpc-network-config-analyzer/pkg/vpcmodel"
+)
+
+func TestVPCResourceModelRegion(t *testing.T) {
+	rc, err := ParseResourcesFromFile(filepath.Join(getTestsDirInput(), "input_experiments_env.json"))
+	require.Nilf(t, err, "err: %s", err)
+	vpcConfigs, err := VPCConfigsFromResources(rc, "", "", nil, false)
+	require.Nilf(t, err, "err: %s", err)
+	vpcConfig := vpcConfigs["crn:17"]
+	require.Equal(t, vpcConfig.VPC.RegionName(), "us-south")
+}
```

Comment:
I would extend this function, to also have input from config json file with transit gateways, 
and check that the generated TransitGateway object in the returned VPCConfig, returns an expected Region object when calling `tgw.Region()`.  Similarly, can check that the VPC object it holds, is as expected with `vpc.Region()`.